ci/jenkins/jobs/projects.yaml

```diff
@@ -237,6 +255,88 @@
           started_status: null
           wrappers: []
           publishers: []
+      - '{name}-{test_name}-for-pull-request':
+          test_name: whole-conformance-pending-label
+          node: null
+          description: 'This is for marking PR as pending for whole conformance test.'
+          builders:
+            - builder-pending-label
+          trigger_phrase: null
+          allow_whitelist_orgs_as_admins: false
+          admin_list: []
+          org_list: []
+          white_list: []
+          only_trigger_phrase: false
+          trigger_permit_all: true
+          status_context: jenkins-whole-conformance
+          status_url: --none--
+          success_status: Pending test. Mark as failure. Add comment /test-whole-conformance to trigger.
+          failure_status: Pending test. Mark as failure. Add comment /test-whole-conformance to trigger.
+          error_status: Pending test. Mark as failure. Add comment /test-whole-conformance to trigger.
```

Comment:
I think we should make it clear that running these tests is not required for the PR to be merged?

Reply:
It makes sense.